docker/Dockerfile.tensorrt

```diff
@@ -0,0 +1,38 @@
+FROM nvcr.io/nvidia/tensorrt:23.04-py3
+
+ENV NVIDIA_DRIVER_CAPABILITIES=compute,video
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive \
+    apt-get -y install \
+    libavfilter-dev \
+    libavformat-dev \
+    libavcodec-dev \
+    libswresample-dev \
+    libavutil-dev\
+    wget \
+    build-essential \
+    ninja-build \
+    cmake \
+    git \
+    python3 \
+    python3-pip \
+    python-is-python3
+
+ARG GEN_PYTORCH_EXT=1
```

Comment:
not needed anymore I assume?

Reply:
oh, it is still used.

Reply:
maybe use the desired extras (in setup.py) as a docker arg? PIP_INSTALL_EXTRAS=samples